shared/templates/create_audit_rules_unsuccessful_file_modification_detailed.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python2
+
+#
+# create_audit_rules_unsuccessful_file_modification_detailed.py
+#        generate template-based checks for unsuccessful file modifications detailed
+#            - audit_rules
+
+
+from template_common import FilesGenerator, UnknownTargetError
+
+import re
+
+class AuditRulesUnsuccessfulFileModificationDetailedGenerator(FilesGenerator):
```

Comment:
Is it possible to to abbreviate part of the class? This it uncanny.

`AuditRulesUFMDetailed` or something like that.